cmd/yalcha-server/main.go

```diff
@@ -15,7 +15,7 @@ func main() {
 		Database: config.DB{
 			Host:     "localhost",
 			Port:     "5432",
-			DBName:   "gis",
+			DBName:   "openstreetmap",
 			User:     "heorhi",
 			Password: "some_password",
```

Comment:
It could be nice to use some env/configuration files for these. It might make it possible to dedup the code in the two ` main.go` files and would make deployments easier to configure in the future.

Reply:
Yes, I'll do it soon.